pkg/internal/apis/acme/types_issuer.go

```diff
@@ -85,6 +85,13 @@ type ACMEIssuer struct {
 	// Defaults to false.
 	// +optional
 	DisableAccountKeyGeneration bool `json:"disableAccountKeyGeneration,omitempty"`
+
+	// Enables requesting a Not After date on certificates that matches the
+	// duration of the certificate. This is not supported by all ACME servers
+	// like Let's Encrypt. If set to true when the ACME server does not support
+	// it it will create an error on the Order.
+	// Defaults to false.
+	EnableDurationFeature bool `json:"enableDurationFeature,omitempty"`
```

Comment:
I don't think these internal types need JSON annotations, but I see that the previous fields have them anyway, so hey ho.

Reply:
The bool above, has an `+optional` annotation, but I don't think that's required here either....but it would be nice to get all these consistent one day.

Reply:
you're right sorry i copy pasted too much